src/components/Menu/MenuItem/__stories__/MenuItem.stories.js

```diff
@@ -72,6 +71,70 @@ export const Label = {
   }
 };
 
+export const SubMenu = {
+  render: () => (
+    <Menu>
+      <MenuItem title="Opens on item hover" onClick={() => console.log(1234)}>
```

Comment:
This `onClick` isn't reproducible, instead it's opening and closing the sub menu

Reply:
are you talking about the menu item that is regular one? this is what it should do according to the original code and what I discussed with Yossi. He said that we don't want to enable onClick on menu item with sub menu

Reply:
by the way i don't won't that this view will stay like this. so after you will approve i will change it to a prettier way - just two menu items- one regular and the other is split

Reply:
Ok
Then I guess it shouldn't open-close the sub menu on click too
![ezgif-3-7a4fbecfa7](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/2c736332-ca89-4007-a511-561bebe5b210)

Reply:
mmmmmm
this is an old behavior from what i remember, checking

Reply:
![Uploading ezgif.com-video-to-gif (15).gif…]()

Reply:
this is the behavior in vibe..